app/uk/gov/hmrc/fileupload/write/envelope/Envelope.scala

```diff
@@ -17,25 +17,32 @@
 package uk.gov.hmrc.fileupload.write.envelope
 
 import cats.data.Xor
+import uk.gov.hmrc.fileupload.controllers.EnvelopeConstraints
 import uk.gov.hmrc.fileupload.write.envelope.Envelope.CanResult
 import uk.gov.hmrc.fileupload.write.infrastructure.{EventData, Handler}
 import uk.gov.hmrc.fileupload.{FileId, FileRefId}
 
 object Envelope extends Handler[EnvelopeCommand, Envelope] {
 
   type CanResult = Xor[EnvelopeCommandNotAccepted, Unit.type]
+  val defaultConstraints = EnvelopeConstraints(100, 25 * 1024 * 1024, 12 * 1024 * 1024)
```

Comment:
Done. The default constraints are moved into uk.gov.hmrc.fileupload.read.envelope.Envelope object